packages/vendure-plugin-accept-blue/src/api/accept-blue-common-resolvers.ts

```diff
@@ -65,19 +74,26 @@ export class AcceptBlueCommonResolver {
     @Ctx() ctx: RequestContext,
     @Parent() orderLine: OrderLine
   ): Promise<AcceptBlueSubscription[]> {
-    // TODO place in service
+    await this.entityHydrator.hydrate(ctx, orderLine, { relations: ['order'] });
+    const subscriptionsForOrderLine =
+      await this.acceptBlueService.subscriptionHelper.getSubscriptionsForOrderLine(
+        ctx,
+        orderLine,
+        orderLine.order
+      );
+    return subscriptionsForOrderLine.map((s) => ({
+      ...s,
+      variantId: orderLine.productVariant.id,
+    }));
+  }
+}
 
-    // await this.entityHydrator.hydrate(ctx, orderLine, { relations: ['order'] });
-    // const subscriptionsForOrderLine =
-    //   await this.stripeSubscriptionService.subscriptionHelper.getSubscriptionsForOrderLine(
-    //     ctx,
-    //     orderLine,
-    //     orderLine.order
-    //   );
-    // return subscriptionsForOrderLine.map((s) => ({
-    //   ...s,
-    //   variantId: orderLine.productVariant.id,
-    // }));
-    throw new Error('Not implemented');
+@Resolver('AcceptBluePaymentMethod')
+export class SavedPaymentMethodsResolver {
+  @ResolveField()
+  __resolveType(value: any): string {
+    return value.payment_method_type === 'card'
```

Comment:
You should be able to put this in the same `AcceptBlueCommonResolver` above, so that we don't need a separate `SavedPaymentMethodsResolver`

Keep in mind that you need a specific order for this to work:
```ts
// This works
@ResolveField('__resolveType')
@Resolver('AcceptBluePaymentMethod')

// While this doesn't
@Resolver('AcceptBluePaymentMethod')
@ResolveField('__resolveType')
```